spec/rubocop/cop/style/trailing_underscore_variable_spec.rb

```diff
@@ -169,6 +169,32 @@
 
         expect(new_source).to eq('a, = foo()')
       end
+
+      context 'with parentheses' do
+        it 'leaves parentheses but removes trailing underscores' do
+          new_source = autocorrect_source('(a, b, _) = foo()')
+
+          expect(new_source).to eq('(a, b,) = foo()')
+        end
+
+        it 'removes parentheses when every assignment is to `_`' do
```

Comment:
I would change the description of this test case and the next to mention that it removes the entire assignment, not just the parentheses. 🙂